app/assets/javascripts/shared/dradis_datatable.js

```diff
@@ -21,8 +54,20 @@ class DradisDatatable {
   unbindDataTable() {
     var that = this;
 
+    that.hideColumns();
```

Comment:
Is this related to the datatable being unbind or should this be called in under the `behaviors()` method?

Reply:
> Is this related to the datatable being unbind or should this be called in under the behaviors() method?

Eh, you're right. Thanks for pointing it out.